src/ray/raylet/node_manager.cc

```diff
@@ -560,7 +560,7 @@ void NodeManager::ProcessNodeManagerMessage(TcpClientConnection &node_manager_cl
 }
 
 void NodeManager::ScheduleTasks() {
-  // This method performs the transition of tasks from PENDING to SCHEDULED.
+  // This method performs the transition of tasks from PENDING to READY.
```

Comment:
Is there a `PENDING` queue? Is `scheduling_policy_.Schedule` moving tasks between queues?

Reply:
there's no `PENDING` queue, because it's a little ambiguous what "pending" means. It could be pending a placement decision, pending object dependencies, or pending dispatch. So we explicitly distinguish between `placeable` (ready for placement decision), `waiting`, and `ready`.

Reply:
It might make sense to update this comment to reflect what you've said, and avoid all caps to avoid confusing pending with an actual queue, and `READY` with the ready queue.